src/index.js

```diff
@@ -16,15 +16,40 @@ document.getElementById('leftParenthesis').addEventListener('click', ()=>updateO
 document.getElementById('rightParenthesis').addEventListener('click', ()=>updateOperation(')'));
 document.getElementById('rest').addEventListener('click', ()=>updateOperation('%'));
 document.getElementById('division').addEventListener('click', ()=>updateOperation('/'));
-document.getElementById('multiplication').addEventListener('click', ()=>updateOperation('x'));
+document.getElementById('multiplication').addEventListener('click', ()=>updateOperation('*'));
 document.getElementById('minus').addEventListener('click', ()=>updateOperation('-'));
 document.getElementById('plus').addEventListener('click', ()=>updateOperation('+'));
 document.getElementById('allClear').addEventListener('click', ()=>allclear());
+document.getElementById('equals').addEventListener('click', ()=>solve())
 
 function updateOperation(operand) {
-    input.value += operand;
+  if(input.value=='' && operand=='%'){
+    alert('You can\'t do that');
```

Comment:
You should not use \ in these cases, you should change the `'` to `"`:

```suggestion
    alert("You can't do that");
```

Reply:
This is a personal preference, but I like to use the quotes ("") more than the ('')